src/declaration.h

```diff
@@ -580,10 +580,10 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
-#ifdef IN_GCC
+#ifdef IN_GCC // %%
```

Comment:
Yeah - upstream need to be aware of any IN_GCC macros - whether or not we can remove these in future is up for discussion.